packages/react-native-version-check/src/getStoreUrl.js

```diff
@@ -29,9 +29,11 @@ export const getAppStoreUrl = async (
       option.country = await getVersionInfo().getCountry();
     }
 
-    return `https://itunes.apple.com/${option.country}/app/${
-      option.appName
-    }/id${option.appID}`;
+    const countryCode = !option.country ? `${option.country}/` : '';
```

Comment:
This will make only invalid codes be inserted

![image](https://user-images.githubusercontent.com/11684628/43357337-d0e8159a-92ba-11e8-879b-1988a5c8248b.png)
![image](https://user-images.githubusercontent.com/11684628/43357346-fd5429b6-92ba-11e8-8cd7-f3ae5e2a6c77.png)